teal/packer/config.tl

```diff
@@ -63,27 +63,25 @@ local default_config: Config = {
   },
   display = {
     non_interactive = false,
-    open_cmd = '65vnew',
     working_sym = '⟳',
     error_sym = '✗',
     done_sym = '✓',
     removed_sym = '-',
     moved_sym = '→',
     item_sym = '•',
     header_sym = '━',
-    show_all_info = true,
     prompt_border = 'double',
     keybindings = {
       quit = 'q',
-      toggle_update = 'u',
-      continue = 'c',
-      toggle_info = '<CR>',
+      toggle_info = {'za', '<CR>'},
       diff = 'd',
       prompt_revert = 'r',
-      retry = 'R',
```

Comment:
I will note that selecting specific plugins for updating and being able to retry updates were very popular features. I think we can do away with selecting plugins for updating given the addition of lockfile support, but keeping retrying around may be more important - this was useful for users on slow internet connections that had timeout issues.

Reply:
Related: one feature I quite like from lazy is the ability to `check` (which basically does `git fetch` instead of `pull` and then shows the log, so shouldn't be hard to add). With this, updating individual plugins that look "safe" might make sense?

Reply:
I've only removed this stuff here to make the refactor easier. Definitely plan to add features around this area. If done properly a lot of these features will hopefully come for free.

`display.lua/tl` was really badly abstracted with too much plugin logic that didn't belong there, so first trying to structure that properly.